filetype.vim

```diff
@@ -9,4 +9,8 @@
 au BufRead,BufNewFile *.rs set filetype=rust
 au BufNewFile,BufRead *.sol setf solidity
 
+" Support for wepy (https://github.com/Tencent/wepy)
+" and layer 'lang#vue' need to be enabled
+au BufRead,BufNewFile *.wpy setlocal filetype=vue.html.javascript.css
```

Comment:
sorry, I can not understand this, why use so long filetype?

Reply:
As *.wpy compose of three parts, including style(css), template(html) and javascript, like vue file,
you can refer to [posva/vim-vue](https://github.com/posva/vim-vue#how-can-i-use-existing-configurationplugins-in-vue-files) for the detail